docs/src/develop.md

```diff
@@ -0,0 +1,58 @@
+# Development notes
+
+## Set up
+
+Clone SpeedyWeather.jl from GitHub and change the current working directory:
+
+```
+git clone https://github.com/milankl/SpeedyWeather.jl.git
+cd SpeedyWeather.jl
+```
+
+To install Julia, manually download the latest version from the [Julia release page](https://julialang.org/downloads/) or, alternately, install it using [juliaup](https://github.com/JuliaLang/juliaup), the official Julia installer and version multiplexer. For Linux, macOS, and WSL run:
+
+```
+curl -fsSL https://install.julialang.org | sh
+```
```

Comment:
I'm still not convinced that our documentation is the place to describe how to install Julia. I believe most users would go to julialang.org or to its documentation